packages/react/src/table/TableScroller.tsx

```diff
@@ -209,11 +209,25 @@ export function TableScroller({ children }: TableScrollerProps) {
 	const hasScroll = thumbWidthRatio !== 1;
 
 	useEffect(() => {
-		setScrollerAriaLabel(
-			`Table ${
-				scrollerRef.current?.querySelector('caption')?.textContent || ''
-			}`
```

Comment:
This seems odd that we were comfortable with `Table ` as our label for the scroller?

Reply:
The section needs to have a name, and since we can't force people to use captions etc, I think all we can do is call it Table